rust/src/file.rs

```diff
@@ -149,7 +147,19 @@ impl File {
             buf.as_mut_slice(),
             Vec3::from(1u32 << block_len_log2),
             self.header.voxel_size as usize,
-            self.header.voxel_type)?;
+            self.header.voxel_type,
+            src_mat.get_is_fortran_order(),
+        )?;
+
+        // build second buffer
+        let mut fortran_conversion_buf = vec![0u8; self.header.block_size()];
```

Comment:
All these buffers start to get a bit confusing (not your fault): There's `self.block_buf` for compression, `buf_mat`, and now `fortran_conversion_buf`. What do you think about using a more consistent naming scheme, like, `src_block_buf` for `buf_mat`, `raw_disk_block_buf` for `fortran_conversion_mat`, and `disk_block_buf` for `block_buf`?

Also, all these buffers are currently allocated and initialized for each visited WKW file. The initialization with zeroes is not really necessary and might slow things down. That's fine for now. But at one point we should profile this and decide whether i) only the buffers that are really needed should be allocated and initialized, ii) buffers should be re-used across reads from / writes to different WKW files within a dataset, and iii) we can (unsafely) skip the zero-initialization.